src/Expensify.js

```diff
@@ -92,9 +92,15 @@ function Expensify(props) {
     const [isNavigationReady, setIsNavigationReady] = useState(false);
     const [isOnyxMigrated, setIsOnyxMigrated] = useState(false);
     const [isSplashHidden, setIsSplashHidden] = useState(false);
+    const [isCheckingPublicRoom, setIsCheckingPublicRoom] = useState(true);
```

Comment:
Let's change the variable name to `hasAttemptedToOpenPublicRoom`, so it can be more descriptive and we can differentiate from the `isCheckingPublicRoom` value from the server.